vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/dataview/GridLazyDataView.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2000-2020 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.vaadin.flow.component.grid.dataview;
+
+import com.vaadin.flow.data.provider.LazyDataView;
+
+/**
+ * Interface for lazy data view for grid.
+ *
+ * @param <T>
+ *            the type of the items in grid
+ */
+public interface GridLazyDataView<T> extends GridDataView<T>, LazyDataView<T> {
```

Comment:
Do we need the empty interface or could it be a class as GridListDataView and just implement the interfaces directly.

Reply:
+1